.travis.yml

```diff
@@ -172,41 +172,13 @@ script:
   # Python3.5+ only. Otherwise we will get `SyntaxError` regardless of how we set the tester.
   - python -c 'import sys;exit(sys.version_info>=(3,5))' || python -m pytest -v --durations=10 python/ray/experimental/test/async_test.py
 
-  - python -m pytest -v --durations=10 python/ray/test/test_global_state.py
-  - python -m pytest -v --durations=10 python/ray/test/test_queue.py
-  - python -m pytest -v --durations=10 python/ray/test/test_ray_init.py
-  - python -m pytest -v --durations=10 test/mini_test.py
-
-  - python -m pytest -v --durations=10 test/runtest.py
-  - python -m pytest -v --durations=10 test/array_test.py
-  - python -m pytest -v --durations=10 test/actor_test.py
-  - python -m pytest -v --durations=10 test/autoscaler_test.py
-  - python -m pytest -v --durations=10 test/tensorflow_test.py
-  - python -m pytest -v --durations=10 test/failure_test.py
-  - python -m pytest -v --durations=10 test/microbenchmarks.py
-  - python -m pytest -v --durations=10 test/stress_tests.py
-  - python -m pytest -v --durations=10 test/component_failures_test.py
-  - python -m pytest -v --durations=10 test/multi_node_test.py
-  - python -m pytest -v --durations=10 test/multi_node_test_2.py
-  - python -m pytest -v --durations=10 test/recursion_test.py
-  - python -m pytest -v --durations=10 test/monitor_test.py
-  - python -m pytest -v --durations=10 test/cython_test.py
-  - python -m pytest -v --durations=10 test/credis_test.py
-  - python -m pytest -v --durations=10 test/node_manager_test.py
+  - python -m pytest -v --durations=10 python/ray/tests
+
   # TODO(yuhguo): object_manager_test.py requires a lot of CPU/memory, and
   # better be put in Jenkins. However, it fails frequently in Jenkins, but
   # works well in Travis. We should consider moving it back to Jenkins once
   # we figure out the reason.
-  - python -m pytest -v --durations=10 test/object_manager_test.py
-
-  # ray temp file tests
-  - python -m pytest -v --durations=10 test/tempfile_test.py
-
-  # ray debug tools tests
-  - python -m pytest -v --durations=10 test/debug_tools_test.py
-
-  # modin test files
-  - python python/ray/test/test_modin.py
+  - python -m pytest -v --durations=10 python/ray/tests/test_object_manager.py
```

Comment:
I think this will run `test_object_manager.py` twice, right? Maybe we can move the above comment to the test_object_manager.py file, and don't need to make this a separate command. cc @guoyuhong

Reply:
I reworded the comment so that I could keep the comment in `.travis.yml` and delete the line so `test_object_manager.py` won't be ran twice